### ISSUE TITLE
[TechDraw] Unable to move section along X axis

### ISSUE THREAD

**Issue by kaiwas** [FreeCAD/FreeCAD#15961]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

The section created on the basis of one of the views cannot be moved horizontally. No settings help.

At the same time, there are values ​​that allow you to move the section on the sheet. But the movement is reset if you move the section with the mouse.

Demonstration on video. 

https://github.com/user-attachments/assets/744acb10-4eab-4a64-92ce-e6f674d5cc7b

Remove the zip extension.
[Test_techDraw.FCStd.zip](https://github.com/user-attachments/files/16657276/Test_techDraw.FCStd.zip)

### Full version info

```shell
OS: Manjaro Linux (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38467 (Git) AppImage
Build type: Release
Branch: main
Hash: 87e73967da5734e100d5ac4b2fd5af1648a02572
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: Russian/Russia (ru_RU)
Installed mods: 
  * OpenTheme 2024.7.30
  * PitchedRoof


```

### Subproject(s) affected?

TechDraw

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
@kaiwas can you please add the Full About info to the ticket ? TIA

**Comment by kaiwas** [comment]
> can you please add the Full About info to the ticket ? TIA

I added information about the current state of the system (is that enough?)
But I noticed this problem more than a month ago.

What is TIA?

**Comment by luzpaz** [comment]
> I added information about the current state of the system (is that enough?)
But I noticed this problem more than a month ago.  

@kaiwas it's just good housekeeping to do so. Sometimes tickets get missed and in this case it would save triage time by knowing what specific revision you tested it on.

> What is TIA?

Thank in advance :grin:

**Comment by WandererFan** [comment]
The fix for this is included in queued PR #15450.

**Comment by kaiwas** [comment]
> The fix for this is included in queued PR #15450.

Good news. Thank you!

**Comment by luzpaz** [comment]
Weird, I can't link this ticket to the PR
